packages/@react-spectrum/s2/src/style-utils.ts

```diff
@@ -13,16 +13,6 @@
 import {CSSProperties} from 'react';
 import {StyleString} from '../style/types';
 
```

Comment:
Should we also expose` getAllowedOverrides()` and `styleProps` from this file?

Reply:
I think we'll keep those internal for now. It's pretty easy to pass in an array to the `style` macro, and different components may have different styles that they want to allow. I exported the StyleString type here so you can use that for now.